tests/test_OrySat.py

```diff
@@ -16,65 +16,57 @@ def test_name(self):
     def test_common_name(self):
         assert self.species.common_name == "Asian rice"
 
-    # QC Tests. These tests are performed by another contributor
-    # independently referring to the citations provided in the
-    # species definition, filling in the appropriate values
-    # and deleting the pytest "skip" annotations.
-    @pytest.mark.skip("Population size QC not done yet")
     def test_qc_population_size(self):
-        assert self.species.population_size == -1
+        assert self.species.population_size == 46875
 
-    @pytest.mark.skip("Generation time QC not done yet")
     def test_qc_generation_time(self):
-        assert self.species.generation_time == -1
+        assert self.species.generation_time == 1
 
 
 class TestGenomeData(test_species.GenomeTestBase):
     genome = stdpopsim.get_species("OrySat").genome
 
```

Comment:
The ploidy test should be an additional function inside this ``TestGenomeData`` class:

```python
 @pytest.mark.parametrize("chrom", [chrom for chrom in genome.chromosomes]) 
 def test_chromosome_ploidy(self, chrom): 
     if chrom.id in ["Mt", "Pt"]: 
         assert chrom.ploidy == 1 
     else: 
         assert chrom.ploidy == 2 
```